components/resources/brave_components_strings.grd

```diff
@@ -990,6 +990,7 @@
       <part file="wallet_strings.grdp" />
       <part file="tor_strings.grdp" />
       <part file="brave_shields_strings.grdp" />
+      <part file="brave_adblock_strings.grdp" />
```

Comment:
Just noticed this, but strings for the settings page go in `app/brave_settings_strings.grdp`. You can probably just move the grdp and reference it from that. This is not a resource for components (and also shouldn't be included on android or ios).

Reply:
done